lib/glimesh/accounts/user.ex

```diff
@@ -290,4 +320,19 @@ defmodule Glimesh.Accounts.User do
       add_error(changeset, :tfa, "Invalid 2FA code")
     end
   end
+
+  @doc """
+  Validates the GCT access level
+  """
+  def validate_gct_level(changeset, needed_amount) do
```

Comment:
What is this exactly? Looks like there's a couple of warnings as well.

Reply:
This makes sure the gct access level is the correct amount of digits long. I changed somewhat how the access levels works and each digit corresponds to a different permission. Right now the 2 digits are: global GCT access and billing access. With the billing access not implemented yet.